docs/src/keps/kep-001.rst

```diff
@@ -0,0 +1,172 @@
+
+Eventually, after discussion, there may be a consensus that the KEP should be accepted – see the next section for details.
+At this point the status becomes ``Accepted``.
+
+Once a KEP has been ``Accepted``, the reference implementation must be completed.
+When the reference implementation is complete and incorporated into the main source code repository, the status will be changed to ``Final``.
+
+To allow gathering of additional design and interface feedback before committing to long term stability for a feature or API, a KEP may also be marked as "Provisional".
+This is short for "Provisionally Accepted", and indicates that the proposal has been accepted for inclusion in the reference implementation, but additional user feedback is needed before the full design can be considered "Final".
+Unlike regular accepted KEPs, provisionally accepted KEPs may still be ``Rejected`` or ``Withdrawn`` even after the related changes have been included in a Klio release.
+
+Wherever possible, it is considered preferable to reduce the scope of a proposal to avoid the need to rely on the "Provisional" status (e.g. by deferring some features to later KEPs), as this status can lead to version compatibility challenges in the wider Klio ecosystem.
+
+A KEP can also be assigned status ``Deferred``.
+The KEP author or a core developer can assign the KEP this status when no progress is being made on the KEP.
+
+A KEP can also be ``Rejected``.
+Perhaps after all is said and done it was not a good idea.
+It is still important to have a record of this fact.
+The ``Withdrawn`` status is similar: it means that the KEP author themselves has decided that the KEP is actually a bad idea, or has accepted that a competing proposal is a better alternative.
+
+When a KEP is ``Accepted``, ``Rejected``, or ``Withdrawn``, the KEP should be updated accordingly.
+In addition to updating the status field, at the very least the ``Resolution`` header should be added with a link to the relevant PR thread.
+
+KEPs can also be ``Superseded`` by a different KEP, rendering the original obsolete.
+The ``Replaced-By`` and ``Replaces`` headers should be added to the original and new KEPs respectively.
+
+Process KEPs may also have a status of ``Active`` if they are never meant to be completed, e.g. KEP 0 (this KEP).
+
+
+How a KEP becomes Accepted
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A KEP is ``Accepted`` by consensus of all interested `core contributors`_.
+We need a concrete way to tell whether consensus has been reached.
+When you think a KEP is ready to accept, make a comment on the PR, pinging `@spotify/klio-devs`_ and asking if there are any objections within 7 days (or longer) from the comment.
+
+If all `@spotify/klio-devs`_ have subsequently approved the PR, or there have been no substantive objections by the given deadline, then the KEP can officially be marked ``Accepted`` by updating the PR and merging.
```

Comment:
This doesn't seem to describe how a KEP can enter the `Active` state, although I imagine the steps are basically the same.

Related to my other comment, maybe it makes sense to describe two separate workflows or more clearly describe which types of PEPs can enter which states.